server/channels/api4/team.go

```diff
@@ -117,6 +117,11 @@ func createTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if team.SchemeId != nil && !c.App.SessionHasPermissionTo(*c.AppContext.Session(), model.PermissionSysconsoleWriteUserManagementPermissions) {
```

Comment:
What if a user has schemeID already set but now in this CreateApi, the user makes it nil then the schemeID would still be updated?

Reply:
Oh, I misunderstood. It's a "create" api so there won't be any existing ID already.